tools/proto-js-plugin/src/main/java/io/spine/js/generate/resolve/ResolveImports.java

```diff
@@ -46,23 +43,13 @@
  *
  * <p>The task should be performed last.
```

Comment:
`last` from which scope? 

Let's describe some context or mention the stages that should be completed by the time this task starts.